Assets/Scripts/Backend/Hidden Classes/RandomizeFindableItems.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using UnityEngine;
+
+internal static class RandomizeFindableItems
+{
+    static RandomizeFindableItems()
+    {
+        LevelApi.OnLevelStart += OnRandomizeFindableItems;
+    }
+
+	private static void OnRandomizeFindableItems(int level)
+	{
+		// In an array you need to get index 0 for level 1.
```

Comment:
This comment raises more questions than it answers:
- Why must the level number be lowered by 1?
- Is this specifically for level 1, like the comment implies?
- Does this mean that levels are ordered by their scene index rather than being manually ordered by preference?